packages/amplify-cli/src/amplify-exception-handler.ts

```diff
@@ -202,6 +205,54 @@ const nodeErrorToAmplifyException = (err: NodeJS.ErrnoException): AmplifyExcepti
   );
 };
 
+type CommonExceptionEntry = {
+  messageIndicator: RegExp;
+  amplifyErrorType: AmplifyErrorType;
+  newMessage?: string;
+  resolution: string;
+};
+
+const commonExceptions: CommonExceptionEntry[] = [
+  {
+    messageIndicator: /The provided token has expired./,
+    amplifyErrorType: 'TokenError',
+    newMessage: 'Your AWS access tokens have expired.',
+    resolution: 'Please refresh your tokens.',
+  },
+  {
+    messageIndicator:
+      /(The AWS Access Key Id you provided does not exist in our records.|The request signature we calculated does not match the signature you provided.|The provided token is malformed or otherwise invalid.)/,
+    amplifyErrorType: 'TokenError',
+    newMessage: 'Your AWS tokens are invalid.',
+    resolution: 'Please refresh your tokens.',
+  },
+  {
+    messageIndicator: /is not authorized to perform/,
+    amplifyErrorType: 'PermissionsError',
+    resolution: 'Please update the permissions.',
```

Comment:
will there be any output that gives the customer an idea about what permissions they should add? Is there an Amplify or AWS docs page we can link to about how to add permissions to a User/Role?